device/globals/game.gd

```diff
@@ -91,6 +97,21 @@ func clicked(obj, pos):
 			get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "hud", "set_tooltip", "")
 
 		elif obj.inventory:
+			# Use and look are the only valid choices with an action menu
+			if action_menu:
+				# Do not set `look` as permanent action
+				if input_event.button_index == BUTTON_RIGHT:
```

Comment:
I don't think `input_event` is declared anywhere.

Reply:
It is `null` by default, but I was not able to cause a problem here. It's as if whenever that code path is traveled, the `input_event` has been passed.